web_src/js/components/RepoBranchTagSelector.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="ui floating filter dropdown custom">
-    <button class="branch-dropdown-button gt-ellipsis ui basic small compact button gt-df gt-m-0" @click="menuVisible = !menuVisible" @keyup.enter="menuVisible = !menuVisible">
+    <button class="branch-dropdown-button gt-ellipsis ui basic small compact button gt-df gt-m-0" @click="menuVisible = !menuVisible;fetchBranches()" @keyup.enter="menuVisible = !menuVisible">
```

Comment:
I think you need to watch the `menuVisible`

Otherwise, you already missed the `@keyup.enter` case here.

Reply:
Updated and this part is not changed.